README.md

```diff
@@ -1,12 +1,12 @@
 # Intake: A general interface for loading data
 
-![Logo](https://github.com/ContinuumIO/intake/raw/master/logo-small.png)
+![Logo](https://github.com/intake/intake/raw/master/logo-small.png)
 
-[![Build Status](https://travis-ci.org/ContinuumIO/intake.svg?branch=master)](https://travis-ci.org/ContinuumIO/intake)
-[![Coverage Status](https://coveralls.io/repos/github/ContinuumIO/intake/badge.svg?branch=master)](https://coveralls.io/github/ContinuumIO/intake?branch=master)
+[![Build Status](https://travis-ci.org/intake/intake.svg?branch=master)](https://travis-ci.org/intake/intake)
```

Comment:
This one breaks - Travis is still watching ContinuumIO, which is probably fine to leave as it is.

Reply:
I don't see it as broken

Reply:
Sorry, must have been some kind of lag